lib/streamlit/hello.py

```diff
@@ -180,53 +258,69 @@ def get_UN_data():
         st.error("Connection Error. This demo requires internet access")
         return
 
-    countries = st.multiselect("Choose countries", list(df.index),
-                               ["China", "United States of America"])
+    countries = st.multiselect(
+        "Choose countries", list(df.index), ["China", "United States of America"]
+    )
     if not countries:
         st.error("Please select at least one country.")
         return
 
-    "### Gross Agricultural Production \[$1B\]"
-    st.write(df.loc[countries].sort_index())
+    st.write("### Gross Agricultural Production ($)", df.loc[countries].sort_index())
+
+    data = df.loc[countries].T.reset_index()
+    data = pd.melt(data, id_vars=["index"]).rename(
+        columns={"index": "year", "value": "Gross Agricultural Product ($)"}
+    )
+    chart = (
+        alt.Chart(data)
+        .mark_area(opacity=0.3)
+        .encode(
+            x="year:T",
+            y=alt.Y("Gross Agricultural Product ($):Q", stack=None),
+            color="Region:N",
+        )
+    )
+    st.write("", "", chart)
+
 
-    ax = df.loc[countries].T.plot(kind="area", figsize=(20, 8), stacked=False)
-    ax.set_ylabel("GAP in Billions (Int $)", fontsize=25)
-    ax.set_xlabel("Year", fontsize=25)
-    ax.tick_params(labelsize=20)
-    ax.legend(loc=2, prop={"size": 20})
-    st.pyplot()
+# fmt: on
 
+DEMOS = OrderedDict(
+    {
+        "---": intro,
+        "Animation Demo": fractal_demo,
+        "Plotting Demo": plotting_demo,
+        "Mapping Demo": mapping_demo,
+        "DataFrame Demo": data_frame_demo,
+    }
+)
 
-DEMOS = OrderedDict({
-    "---": intro,
-    "Mapping Demo": mapping_demo,
-    "Animation Demo": fractal_demo,
-    "DataFrame Demo": data_frame_demo,
-    "Plotting Demo": plotting_demo,
-})
 
 def run():
     demo_name = st.sidebar.selectbox("Choose a demo", list(DEMOS.keys()), 0)
     demo = DEMOS[demo_name]
 
-    if demo_name != "---":
-        show_code = st.sidebar.checkbox("Show code", True)
-        st.markdown("# %s" % demo_name)
-        st.write(inspect.getdoc(demo))
-        demo()
-        if show_code:
-            st.markdown("## Code")
-            sourcelines, n_lines = inspect.getsourcelines(demo)
-            sourcelines = reset_indentation(remove_docstring(sourcelines))
-            st.code("".join(sourcelines))
-    else:
-        st.title("Welcome to Streamlit!")
+    if demo_name == "---":
+        show_code = False
         st.write(
             """
-            Streamlit is the best way to create bespoke apps.
+            # Welcome to Streamlit
+            ## The fastest way to build custom ML tools
             """
         )
-        demo()
+    else:
+        show_code = st.sidebar.checkbox("Show code", True)
+        st.markdown("# %s" % demo_name)
+        st.write(inspect.getdoc(demo))
+        # Clear everything from the intro page.
+        for i in range(10):
```

Comment:
why is this correct? Do we have 10 elements? Seems like magic to me. Maybe add a comment to explain why.

Reply:
Added a comment.